spec/unit/crypto/algorithms/megolm.spec.js

```diff
@@ -68,6 +68,9 @@ describe("MegolmDecryption", function() {
             // 'crypto' implementation.
```

Comment:
> I've resolved now all the points and would be very happy to see this finally merged 🙂

For reference, I'm not an Element-Web maintainer to merge. Just gave it a review glance 🙂